### ISSUE TITLE
Problem with launching some Epic games

### ISSUE THREAD

**Issue by George22x** [Alia5/GlosSI#188]
There are some games on Epic Store that cannot be launched if they are added to Steam as a non-Steam game (Borderlands 2 is an example) and must be run directly from the Epic Launcher. This is probably due to DRM. If I add such game to Steam via GlosSI, the result is the same. Could a feature be added to GlosSI to make them run? I tested it with the latest snapshot-build GlosSI 0.1.0.2 with no success.

## General Information

  - Version: GlosSI 0.1.0.2
  - Platform: Win10
  - Controller(s): 1 DS4 Controller
  - Application: GlosSIConfig
  
## Logs / Additional Files

(Log and config files are located in `%appdata%/GlosSI`)

**Comment by George22x** [comment]
To be more specific about the issue: many Epic games run just fine as non-Steam game. Howewer, some do not due to some kind of protection. This list shows DRM-free plus unverified/DRM games and some of these protected games may have problems to run (I encountered them for example with Tiny Tina's Assault on Dragon Keep, Borderlands 2, Borderlands: The Pre-Sequel and The Long Dark):
https://www.pcgamingwiki.com/wiki/List_of_DRM-free_games_on_Epic_Games_Store
There are various workaround advices that can be found on the net, for example:
https://steamcommunity.com/sharedfiles/filedetails/?id=2476375614
But I found no reliable method so far that would work for me.
